transports/sql/design.md

```diff
@@ -37,6 +37,14 @@ The `ReplyToAddress` column contains the value of `NServiceBus.ReplyToAddress` h
 The `Recoverable` column should always contain the value `1` to ensure wire-level compatibility with transport Version 2 and lower.
 
 
+### Backwards Compatibility
+
+The `CorrelationId`, `ReplyToAddress` and `Recoverable` columns are required for backwards compatibility with version 1 of the [NServiceBus.SqlServer](https://www.nuget.org/packages/NServiceBus.SqlServer) transport
+
+include: backwards-compatibility-receive-message
+
+include: backwards-compatibility-send-message
+
```

Comment:
In the "how it's designed" document we don't need all the details of what goes where.
```suggestion
```